src/main/kotlin/fi/metatavu/metaform/server/controllers/AdminThemeController.kt

```diff
@@ -0,0 +1,75 @@
+package fi.metatavu.metaform.server.controllers
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import fi.metatavu.metaform.server.persistence.dao.AdminThemeDAO
+import fi.metatavu.metaform.server.persistence.model.AdminTheme
+import java.util.UUID
+import javax.enterprise.context.ApplicationScoped
+import javax.inject.Inject
+
+/**
+ * Controller for admin themes
+ */
+@ApplicationScoped
+class AdminThemeController {
+    @Inject
+    lateinit var adminThemeDAO: AdminThemeDAO
+
+    private val mapper = ObjectMapper()
```

Comment:
```suggestion

    @Inject
    lateinit var mapper: ObjectMapper
```
using a injected mapper is better than creating one

Reply:
Changed.